admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -426,17 +430,43 @@ const featuresCollection = (function () {
         return url.replace(/\/\d+(?!.*\/\d+)((?=\?.*))?/, `/${newId}`);
       }
 
+      // adding events to the preloaded features
+      $(document).on('change', '.feature-collection select.feature-value-selector', (event) => {
+        const that = event.currentTarget;
+        const $row = $($(that).parents('.row')[0]);
+
+        const $customValueSelector = $row.find('input[type=text]');
+        $customValueSelector.val('');
+      });
+
+      $(document).on('keyup', '.feature-collection input[type=text]', (event) => {
+        const that = event.currentTarget;
+        const $row = $($(that).parents('.row')[0]);
```

Comment:
@NeOMakinG should we handle the case where there is no offset zero? should we not verify the DOM structure?

Reply:
Is there a case where the dom structure would be different? I don't think so, this is why I didn't comment on this

But if you want to verify, feel free, shouldn't add too much pain and problems